content/03-reference/01-tools-and-interfaces/02-prisma-client/05-filtering.mdx

```diff
@@ -0,0 +1,323 @@
+---
+title: 'Filtering'
+metaTitle: 'Filtering (Reference)'
+metaDescription: 'Advanced filtering use cases.'
+---
+
+## Overview
+
+Prisma Client allows you to filter records on any combination of model fields, including related models, and supports a variety of filter conditions.
+
+For example, the following query returns all `User` records with at least one published post and an email address that ends with `prisma.io`:
+
+```javascript
```

Comment:
To save some keystrokes (and overal consistency), you can use `js` as the language annotionation instead of `javascript` (as well as `ts` for `typescript`)

Reply:
Done! ✅